### ISSUE TITLE
Annotating action descriptors

### ISSUE THREAD

**Issue by kiragrif** [amrisi/amr-guidelines#157]
Occasionally we run into things that are "action descriptors" (i.e. things that supposedly describe actions of the author)

Currently we're just treating these as regular text:
```
Snt 29 in workset  dfb-0516
*shaking head in disgust*

(s / shake-01
      :ARG1 (h2 / head)
      :manner (d / disgust-01))
```
Do we want to somehow indicate that this is an extra-conversational descriptor / an actual action of the post author?

**Comment by nschneid** [comment]
Nonverbal speech acts! Maybe `:mode expressive` or `:mode nonverbal`??

**Comment by nschneid** [comment]
To clarify, do you ever have narrated actions that are *not* speech acts? E.g.: 

    I wonder how much it costs... *walks over to check the menu*

**Comment by cbonial** [comment]
It would be nice to do something along the lines of how we have treated vocatives, where authors address their comments to specific listeners.  In those cases, we introduce "say" and the appropriate unmentioned entities in the role slots.  Here, perhaps we could tie these to whatever the author is reacting to (e.g., Trump leading the polls?! *shaking head in disgust* -- where Trump leading the polls would be the cause of the author's ("I") shaking head in disgust) This would depend upon how the text is sliced into annotation instances -- are these generally distinct units, or are they embedded in a larger annotation instance?

**Comment by kiragrif** [comment]
>Nonverbal speech acts!

Most of these are nonverbal, but I found one that is verbal:
```
dfb-0107
*Note to self: Call Mom and thank her for teaching me the ability to think for myself.
```

> Maybe :mode expressive or :mode nonverbal??

What about `:mode narrative` since you had also described them as narrated actions?

**Comment by kiragrif** [comment]
> Here, perhaps we could tie these to whatever the author is reacting to (e.g., Trump leading the polls?! shaking head in disgust -- where Trump leading the polls would be the cause of the author's ("I") shaking head in disgust) 

Yes, agreed this would be good to do.

> This would depend upon how the text is sliced into annotation instances -- are these generally distinct units, or are they embedded in a larger annotation instance?

They vary.

Sometimes they are distinct units, in which case we wouldn't be able to connect them to their instigating predicate (unless we did cross-sentence annotation):
```
dfb-0331
*runs back to lurkdom*

dfb-0073
*crickets*

dfb-0104
*Laughs*
```

But sometimes they're embedded in a larger SU, in which case we could connect them to what the author is reacting to:
```
dfb-0320
*drool* She may be a socialist, but at least she's a hot one!

dfb-0451
*shrugs* They have a chance at getting elected just as much as the next party.

dfb-0461
*snort* best product E.V.A.R. LMFAO
```

These can also be chained together in a single SU, where they may have an implied causal (or at least temporal) relationship:
```
fb-0489
*pan* *hilarious Dawkins face* *laughter*
```

**Comment by ulfulf** [comment]
I like Claire's reference to vocatives where we add a ```say-01```:
```
Dad, I need more money.
(s / say-01 
  :ARG0 i 
  :ARG1 (n / need-01 
          :ARG0 (i / i) 
          :ARG1 (m / money 
                  :mod (m2 / more))) 
  :ARG2 (p / person 
          :ARG0-of (h / have-rel-role-91 
                     :ARG1 i 
                     :ARG2 (f / father)))) 
```

The non-speech acts that Kira mentions are somewhat the opposite. In a context of a speaker saying somebody to an audience (where the *say-01* might be implied), there is a non-saying. In a sense, the "implied say-01" mode is temporarily paused.

```
[From Romeo and Julia]
MONTAGUE: I would thou wert so happy by thy stay,
	      To hear true shrift. Come, madam, let's away.
(Exeunt MONTAGUE and LADY MONTAGUE)
```

```
[From a psychology journal paper]
J.G.P.: ... I'm very sorry. Spiders. Oh. (Shakes head in disgust.)	
```

A slightly wild idea might be to annotate this as something like:
```
*shaking head in disgust*					 
(s / shake-01
  :ARG1 (h2 / head)
  :manner (d / disgust-01)
  :ARG1-of (s2 / say-01 :polarity -))
```

**Comment by nschneid** [comment]
Interesting thought. But what about "She said it with her eyes"? I don't know that we want to assume that saying is always verbal.

**Comment by timjogorman** [comment]
I totally agree that the transcript/play examples access this unstated top-level "say-01", but I'm not convinced that these more "expressive" action depictions do; for one thing, you can embed them inside the AMR, like:

> sarah just texted me: *snort* best product E.V.A.R. LMFAO

I'd be happy with something just reserved for that class of terms, like ":mode action-depiction". 

On a higher level, I wish that the ":mode expressive" thing was a more clearly defined function.  When I see :mode expressive for "shoot!" I interpret it as meaning  "someone is expressing the connotations evoked by saying "shoot!", rather than expressing the actual event of shooting".  In that sense, we could easily have done them in a way that allowed action depictions, as in:

> shoot!
```
expressive-91
   :arg1 say-01
       :arg1 shoot-01
```
> lol
```
expressive-91
   :arg1 laugh-01
       :manner out-loud
```
> *drool*
```
expressive-91
   :arg1 drool-01
```

**Comment by nschneid** [comment]
>When I see :mode expressive for "shoot!" I interpret it as meaning  "someone is expressing the connotations evoked by saying "shoot!", rather than expressing the actual event of shooting".  In that sense, we could easily have done them in a way that allowed action depictions, as in:
>
>> shoot!
>>```
>>expressive-91
>>   :arg1 say-01
>>       :arg1 shoot-01
>>```

For me, the interjection "shoot!" has no more connection to shooting than "darn!" has to socks. I'd just say

```
(s / shoot :mode expressive)
```

**Comment by nschneid** [comment]
My friend pointed me to a master's thesis on these! [Bomersbach](http://d-scholarship.pitt.edu/7127/) calls them "textually described actions". How about `:mode action`?
